silkworm/node/stagedsync/stages/stage_senders.cpp

```diff
@@ -287,10 +287,8 @@ Stage::Result Senders::parallel_recover(db::RWTxn& txn) {
                                                                   " > target progress " + std::to_string(target_block_num));
         }
 
-        secp256k1_context* context = secp256k1_context_create(SILKWORM_SECP256K1_CONTEXT_FLAGS);
+        static secp256k1_context* context = secp256k1_context_create(SILKWORM_SECP256K1_CONTEXT_FLAGS);
```

Comment:
I don't see a problem with keeping the context forever because we need it regularly to recover senders of new transactions. `static` is fine because the context is safe to use (read-only) from multiple threads.

Reply:
Fair enough, but to address this properly we need to add a leak sanitizer to our CI. I'd rather keep the current PR as is because it's not introducing a new pattern – `static secp256k1_context* context` is already used in other places like `Transaction::sender()`.

Reply:
I like the solution with `SecP256K1Context` wrapper, but I'm going to merge this PR as is and open a new issue for adding leak sanitizer in our CI. Then, we can do some code refactoring to use `SecP256K1Context` as a pattern.

Reply:
Thinking about it again, a new issue is not needed because we already have leak detection in our CI: it's enabled in the CI Fuzzer Tests workflow which runs weekly.